roles/shiftstack/defaults/main.yml

```diff
@@ -35,3 +37,4 @@ cifmw_shiftstack_run_playbook: "ocp_testing.yaml"
 cifmw_shiftstack_sc: "local-storage"
 cifmw_shiftstack_shiftstackclient_artifacts_dir: "/home/cloud-admin/artifacts"
 cifmw_shiftstack_shiftstackclient_installation_dir: "{{ cifmw_shiftstack_shiftstackclient_artifacts_dir }}/installation"
+cifmw_shiftstack_shiftstackclient_incluster_kubeconfig_dir: "/home/cloud-admin/kubeconfig"
```

Comment:
Don't you think this filename could not be descriptive enough?

Reply:
Modified